multisensor/src/main/java/com/example/upm/androidthings/driversamples/MultiSensorActivity.java

```diff
@@ -0,0 +1,205 @@
+/*
+ * Copyright (c) 2017 Intel Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.upm.androidthings.driversamples;
+
+import android.app.Activity;
+import android.os.AsyncTask;
+import android.os.Bundle;
+import android.util.Log;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.example.upm.androidthings.driversupport.BoardDefaults;
+
+import mraa.mraa;
+
+public class MultiSensorActivity extends Activity {
+    private static final String TAG = "MultiSensorActivity";
+
+    short[][] rgb = new short[][]{
+            {0xd1, 0x00, 0x00},   // red
+            {0x33, 0xdd, 0x00},   // green
+            {0x11, 0x33, 0xcc}};   // blue
+    TextView tv;
+    ImageView im1;
+    ImageView im2;
+    ImageView im3;
+
+    upm_tmp006.TMP006 thermopile;
+    upm_jhd1313m1.Jhd1313m1 lcd;
+    upm_grove.GroveLed Redled;
+    upm_grove.GroveButton button;
+    upm_grove.GroveLed Blueled;
+    upm_ttp223.TTP223 touch;
+    upm_grove.GroveLed Greenled;
+
+    float Temp = 0.0f;
+    private int ndx = 0;
+    Runnable thermopileTask = new Runnable() {
+
+        @Override
+        public void run() {
+            // Moves the current thread into the background
+            android.os.Process.setThreadPriority(android.os.Process.THREAD_PRIORITY_BACKGROUND);
+
+            thermopile.setActive();
+            try {
+                if (lcd != null) {
+                    lcd.clear();
+                }
+                while (true) {
+                    // Print out temperature value in °C
+                    Temp = thermopile.getTemperature(1);
+                    Log.i(TAG, "Temperature: " + Temp + " °C");
+
+                    updateUI();
+                    Thread.sleep(1000);
+                }
+            } catch (InterruptedException e) {
+                Thread.currentThread().interrupt();
+            } finally {
+                lcd.delete();
+                thermopile.delete();
+                Redled.off();
+                Redled.delete();
+                button.delete();
+                Blueled.off();
+                Blueled.delete();
+                touch.delete();
+                Greenled.off();
+                Greenled.delete();
+                MultiSensorActivity.this.finish();
+            }
+        }
+    };
+
+    @Override
```

Comment:
When i did a reformat code from android studio, there was a re-arrange code checkbox checked in and so the run task is moved before oncreate(). This works fine as oncreate() is the first method to be executed. Tested the applications.
Also maintained consistency with all the other applications as well. As discussed with you, this code is good to be merged :)